src/main/java/com/sucy/skill/dynamic/trigger/ConsumeTrigger.java

```diff
@@ -30,6 +30,11 @@ public String getKey() {
     public boolean shouldTrigger(PlayerItemConsumeEvent event, int level, Settings settings) {
         final String item_name = event.getItem().getType().name();
         final String req = settings.getString("material", "Any").replace(" ","_");
+        if (item_name.equalsIgnoreCase("potion") && req.equalsIgnoreCase(item_name)){
+            PotionMeta potion = (PotionMeta) event.getItem().getItemMeta();
+            String type = potion.getBasePotionData().getType().getEffectType().getName();
+            if (!settings.getString("potion", "Any").replace(" ","_").equalsIgnoreCase(type)) return false;
```

Comment:
This will fail if the potion is put in as 'Any'. Any will never equal the used potion type and will always return false.